doc/dasd.md

```diff
@@ -0,0 +1,75 @@
+
+DASDs
+-----
+
+There are ECKD and FBA DASDs.
```

Comment:
NP: I was wondering what all these abbreviations mean, later I noticed that there is a glossary at the bottom. Mabey you could add a note like `(See the glossary at the bottom.)`